mods/ctf/ctf_map/ctf_map_core/schem_map.lua

```diff
@@ -214,7 +214,7 @@ math.randomseed(os.time())
 
 -- Fisher-Yates shuffling algorithm, used for shuffling map selection order
 -- Adapted from snippet provided in https://stackoverflow.com/a/35574006
-local function shuffle_maps()
+local function shuffle_maps(not_first)
```

Comment:
Variable name `not_first` is confusing at first read, and a better name would be nice.

Reply:
Any suggestions?

How about `previous_idx`?

Reply:
Hmmm... maybe `idx_to_avoid`? It's a little too verbose, but one would immediately understand what it is.